airflow/gcp/operators/display_video.py

```diff
@@ -0,0 +1,301 @@
+
+    :param report_id: Report ID to delete.
+    :type report_id: str
+    :param api_version: The version of the api that will be requested for example 'v3'.
+    :type api_version: str
+    :param gcp_conn_id: The connection ID to use when fetching connection info.
+    :type gcp_conn_id: str
+    :param delegate_to: The account to impersonate, if any. For this to work, the service accountmaking the
+        request must have  domain-wide delegation enabled.
+    :type delegate_to: str
+    """
+
+    template_fields = ("report_id",)
+
+    @apply_defaults
+    def __init__(
+        self,
+        report_id: str,
+        api_version: str = "v1",
+        gcp_conn_id: str = "google_cloud_default",
+        delegate_to: Optional[str] = None,
+        *args,
+        **kwargs
+    ) -> None:
+        super().__init__(*args, **kwargs)
+        self.report_id = report_id
+        self.api_version = api_version
+        self.gcp_conn_id = gcp_conn_id
+        self.delegate_to = delegate_to
+
+    def execute(self, context: Dict):
+        hook = GoogleDisplayVideo360Hook(
+            gcp_conn_id=self.gcp_conn_id,
+            delegate_to=self.delegate_to,
+            api_version=self.api_version,
+        )
+        self.log.info("Deleting report with id: %s", self.report_id)
+        hook.delete_query(query_id=self.report_id)
+        self.log.info("Report deleted.")
+
+
+class GoogleDisplayVideo360DownloadReportOperator(BaseOperator):
+    """
+    Retrieves a stored query.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:GoogleDisplayVideo360DownloadReportOperator`
+
+    .. seealso::
+        Check also the official API docs:
+        `https://developers.google.com/bid-manager/v1/queries/getquery`
+
+    :param report_id: Report ID to retrieve.
+    :type report_id: str
+    :param bucket_name: The bucket to upload to.
+    :type bucket_name: str
+    :param report_name: The report name to set when uploading the local file.
+    :type report_name: str
+    :param chunk_size: File will be downloaded in chunks of this many bytes.
+    :type chunk_size: int
+    :param gzip: Option to compress local file or file data for upload
+    :type gzip: bool
+    :param api_version: The version of the api that will be requested for example 'v3'.
+    :type api_version: str
+    :param gcp_conn_id: The connection ID to use when fetching connection info.
+    :type gcp_conn_id: str
+    :param delegate_to: The account to impersonate, if any. For this to work, the service accountmaking the
+        request must have  domain-wide delegation enabled.
+    :type delegate_to: str
+    """
+
+    template_fields = ("report_id", "bucket_name", "report_name")
+
+    @apply_defaults
+    def __init__(
+        self,
+        report_id: str,
+        bucket_name: str,
+        report_name: str,
+        gzip: bool = False,
```

Comment:
Can we default compression to True as these reports can get quite large?
```suggestion
        gzip: bool = False,
        gzip: bool = True,
```